src/components/title-list-item/title-list-item.js

```diff
@@ -21,7 +22,12 @@ export default function TitleListItem({
       })}
     />
   ) : (
-    <Link to={link} className={styles.item}>
+    <Link
+      to={link}
+      className={cx('item', {
+      'is-selected': active
+    })}
```

Comment:
Odd formatting here too

Reply:
I thought this format is more readable...I can change it back to a one-liner

Reply:
I'm referring to the spacing.

```
      className={cx('item', {
      'is-selected': active
    })}
```

vs



```
      className={cx('item', {
        'is-selected': active
      })}
```

Reply:
My editor does this to me all the time and I have to manually align them with spaces